packages/ui-components/src/lib/components/detail/OrderDetail.svelte

```diff
@@ -91,6 +99,64 @@
 			errToast('Failed to refresh');
 		}
 	};
+
+	const vaultTypes: { key: string; type: RaindexVaultType }[] = [
+		{ key: 'Output vaults', type: 'output' },
+		{ key: 'Input vaults', type: 'input' },
+		{ key: 'Input & output vaults', type: 'inputOutput' }
+	];
+	const getDefaultVaultsGroupedByType = (): VaultsGroupedByType => ({
+		output: [],
+		input: [],
+		inputOutput: []
+	});
+
+	$: vaultsGroupedByTypes =
```

Comment:
i don't get why we need to group here then later do `vaultsGroupedByTypes[type]` - wasn't filtering the array by type (as we did inline before) more straightforward?